### PR TITLE
Changes for week of Nov 18

### PR DESCRIPTION
**Pull request by duglin**
- Allow for ?inline (same as ?inline=*) and clarify the text a bit
- Don't allow `^` in ifvalue "VALUE" - we'll use this for regexp detection in the future
- Add text in the Primer about xref and how deleting the targets might result in dangling pointers - impl choice if they want to do something about it
- And provide some guidance on choosing proper groups for resources
- Add ETag/If-Match semantics (leveraging `epoch` value)

Fixes: #25
Fixes: #135

### CODE REVIEW

File: core/spec.md [hunk 1]
```diff
@@ -1906,6 +1906,8 @@ The following describes the attributes of Registry model:
     level, are active at the same time then there MUST NOT be duplicate
     `ifvalues` attributes names between those `ifvalues` sections.
   - `ifvalues` `"VALUE"` MUST NOT be an empty string.
+  - `ifvalues` `"VALUE"` MUST NOT start with the `^` (carrot) character as its
```
Comment (jskeet) [inline core/spec.md:1933]: carrot => caret

File: core/spec.md [hunk 2]
```diff
@@ -5034,19 +5036,27 @@ inline[=PATH[,...]]
 
 Where `PATH` is a string indicating which inlinable attributes to show in
 in the response. References to nested attributes are represented using a
-dot(`.`) notation - for example `GROUPS.RESOURCES`. To reference an attribute
-with a dot as part of its name, the JSON PATH escaping mechanism MUST be
-used: `['my.name']`. For example, `prop1.my.name.prop2` would be specified
-as `prop1['my.name'].prop2` if `my.name` is the name of one attribute.
+dot(`.`) notation where the xRegistry collections names along the hierarchy
```
Comment (jskeet) [inline core/spec.md:5072]: Space after "dot"?

File: core/spec.md [hunk 3]
```diff
@@ -5034,19 +5036,27 @@ inline[=PATH[,...]]
...
+are concatenated. For example: `endpoints.messages.versions` will inline all
+Versions of Messages. Non-leaf parts of the `PATH` MUST only reference
+xRegistry collection names and not any specific entity IDs since `PATH` is
+meant to be an abstract traversal of the model.
+
+To reference an attribute with a dot as part of its name, the JSON PATH
```
Comment (jskeet) [inline core/spec.md:5078]: "JSON PATH" => "JSONPath"?

**Review by jskeet — COMMENTED** [review]
Just a brief review

File: core/spec.md (hunk 2 above)
Comment (duglin) [reply core/spec.md:5072]: done

File: core/spec.md (hunk 3 above)
Comment (duglin) [reply core/spec.md:5078]: done

**Comment by duglin** [comment]
Approved on the 11/21 call

**Pull request merged**